### ISSUE TITLE
Download v2 und Upgrade?

### ISSUE THREAD

**Issue by devaxel** [krabina/Datencockpit#50]
Hi,
laut Website ist die V2 nun verfügbar, aber ich kann dessen Download auf Github nicht finden. Bei "Releases" endet es es nach wie vor mit v1.

**Comment by krabina** [comment]
ja, ich bin noch nicht dazugekommen, ein Release zu machen. Siehe auch https://github.com/krabina/Datencockpit/issues/49
